### PR TITLE
ECC-1500: Move VAT Group page

### PR DESCRIPTION
**Pull request by JakeGoodwin**

### CODE REVIEW

**Review by RobertLaverick — CHANGES_REQUESTED** [review]
There is a TODO which should probably be removed/replaced but the only thing I'd ask is if you can clean up the compile warnings before you merge please

File: app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/VatGroupController.scala [hunk 1]
```diff
@@ -18,31 +18,58 @@ package uk.gov.hmrc.eoricommoncomponent.frontend.controllers
...
+          yesNoAnswer =>
+            subscriptionDetailsService.cacheVatGroup(yesNoAnswer).flatMap {
+              _ =>
+                if (isInReviewMode && yesNoAnswer.isNo)
+                  Future.successful(Redirect(VatDetailsController.reviewForm(service)))
+                else if (yesNoAnswer.isNo) Future.successful(Redirect(VatDetailsController.createForm(service)))
+                else
+                  Future.successful(
+                    Redirect(routes.VatGroupsCannotRegisterUsingThisServiceController.form(service))
+                  ) // TODO: Continue new VAT journey
```
[the remark this answers is not in the document]
Comment (JakeGoodwin) [reply app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/VatGroupController.scala:70]: Yep this is a placeholder for where we expect the new redirects to be added once the page exists :)

[the remark this answers is not in the document]
Comment (RobertLaverick) [reply app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/VatGroupController.scala:70]: Figured as much

**Review by RobertLaverick — APPROVED** [review]
For the record we should avoid throwing exceptions if we can, but I won't stop the PR for that.

**Pull request merged**